documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -135,8 +135,21 @@ spec:
 EOF
 ----
 <1> Specify the name of the VMware `Provider` CR.
-<2> Specify the managed object reference (MOR) of the VMware host.
+<2> Specify the managed object reference (MOR) of the VMware host. To retrieve the name of the MOR, see the note that follows.
 <3> Specify the IP address of the VMware migration network.
++
+[NOTE]
+====
+To retrieve the name of the MOR:
+
+. Open a browser and navigate to `https://<url_of_vCenter_server_with_VMware_host>/mob`.
+. Log in when prompted.
+. Select the content link.
+. Select the `rootFolder`.
+. Select the `childEntity` datacenter object type.
++
+The datastores, including the MOR, are displayed.
```

Comment:
and then generalize this part as well - we see there the MOR of other entities as well

Reply:
@ahadas Do you mean write "The datastores including the MORs, are displayed." ?

Reply:
nope, I meant to replace this with something like "In this screen, the MOR of datastores, networks and other vSphere entities, are displayed.

Reply:
```suggestion
From this screen you can navigate to entities within the datacenter and find their MOR.
```